src/edge/simulator.ts

```diff
@@ -223,13 +225,44 @@ export class Simulator {
         });
     }
 
+    private async checkAndGuideUserToSetupIotedgehubdev(outputChannel: vscode.OutputChannel) {
+        if (await this.isValidateConfigSupported()) {
+            try {
+                await Executor.executeCMD(null, "iotedgehubdev", { shell: true }, "validateconfig");
+            } catch (error) {
```

Comment:
will we check error code?

Reply:
If we have error code, do we need is versionsupported ?

Reply:
The error code is always -1 for current version